mycodo/controllers/controller_trigger.py

```diff
@@ -121,28 +121,31 @@ def loop(self):
             check_approved = False
 
             # Check if the trigger period has elapsed
-            if self.trigger_type in ['trigger_sunrise_sunset',
-                                     'trigger_run_pwm_method']:
```

Comment:
This was also executed for run_pwm_method and messed up its period. Have split up this into two separate if branches